crates/runtime/src/builder.rs

```diff
@@ -133,9 +142,18 @@ impl RuntimeBuilder {
             None => status::RuntimeStatus::new(),
         };
 
+        let background_handle = self
+            .tokio_background_runtime
+            .clone()
+            .unwrap_or_else(|| Handle::current());
```

Comment:
Could falling back to `Handle::current()` cause issue here somehow?

Reply:
The fallback is essentially the current behavior